spec/rails_helper.rb

```diff
@@ -10,6 +10,15 @@
 require 'rspec/rails'
 # Add additional requires below this line. Rails is not loaded until this point!
 
+FollowerFollowed.destroy_all
+UserLike.destroy_all
+Post.destroy_all
+User.destroy_all
+ActiveRecord::Base.connection.reset_pk_sequence!('follower_followeds')
+ActiveRecord::Base.connection.reset_pk_sequence!('user_likes')
+ActiveRecord::Base.connection.reset_pk_sequence!('posts')
+ActiveRecord::Base.connection.reset_pk_sequence!('users')
+
```

Comment:
In switching from `sqlite3` to `postgresql`, the `.create!` methods throughout the tests meant that the tables were filling up, messing up tests.  So now, with this, anything `.create!`'d will be destroyed with every run of the test, and their ids will be reset